### ISSUE TITLE
v0.18.0 breaks WASM compatibility

### ISSUE THREAD

**Issue by Smona** [SamiPerttu/fundsp#43]
We are considering using fundsp to build an audio engine for a browser-based DAW. v0.17.1 was looking very promising, but when we upgrade to v0.18.0, our build starts to fail with:

```
  --> /home/smona/.cargo/registry/src/index.crates.io-6f17d22bba15001f/no_denormals-0.1.2/src/lib.rs:24:1
   |
24 | compile_error!("This crate only supports x86, x86_64 and aarch64.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Would it be possible to gate usage of `no_denormals` behind a feature, or otherwise continue supporting the `wasm32-unknown-unknown` target?

**Comment by SamiPerttu** [comment]
Hi! Yes, absolutely. I didn't realize it fails to compile on some targets. I removed `no_denormals` altogether for now and released version 0.18.1.

**Comment by Smona** [comment]
Wow, that was fast! Seeing this level of support we are very excited to move forward with fundsp and check out the improvements in 0.18. Thank you for the amazing library!!
